### PR TITLE
Add fromsource, ns filters in system policy

### PR DESCRIPTION
**Pull request by seswarrajan**
Signed-off-by: Eswar Rajan Subramanian <eswar@accuknox.com>

### CODE REVIEW

File: src/config/configManager.go [hunk 1]
```diff
@@ -176,6 +176,19 @@ func LoadConfigFromFile() {
...
 
+	var sys_ns, sys_notNs []string
+	sys_namespaces := viper.GetStringSlice("application.system.namespace-filter")
+	for _, n := range sys_namespaces {
+		if n[0] == '!' {
+			sys_notNs = append(sys_notNs, n[1:])
+		} else {
+			sys_ns = append(sys_ns, n)
+		}
+	}
```
Comment (nyrahul) [inline src/config/configManager.go]: Can we refactor the code ... between application and network? Seems we can create a function to read the filter and return the ns and notNs vars.

File: src/plugin/kubearmor.go [hunk 2]
```diff
@@ -432,6 +456,26 @@ func StartKubeArmorRelay(StopChan chan struct{}, cfg types.ConfigKubeArmorRelay)
...
+						}
+					}
+				}
+
+				for _, srcFilter := range fromSourceFilter {
+					if strings.Contains(res.Source, srcFilter) {
+						continue
+					}
+				}
+
```
Comment (nyrahul) [inline src/plugin/kubearmor.go]: This code seems redundant to what is available before. Can you refactor and create a function to handle this?

**Review by nyrahul — CHANGES_REQUESTED** [review]

**Review by wazir-ahmed — APPROVED** [review]
Other than @nyrahul's comment, I don't have anything new to add. The PR looks good to me.

**Review by nyrahul — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**